kubernetes/samples/charts/traefik/README.md

```diff
@@ -91,6 +91,80 @@ TLS_PORT=`kubectl -n traefik get service traefik-operator -o jsonpath='{.spec.po
 $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:${TLS_PORT}/testwebapp/
 ```
 
+## SSL Termination at Ingress Controller
```

Comment:
SSL Termination at Ingress Controller -> SSL termination at ingress controller (we use sentence capitalization)

Reply:
done